_pages/API/01-general.md

```diff
@@ -270,13 +270,55 @@ Content-Type: application/json; charset=utf-8
 if your chosen tool or script language does not work directly with present encoding.
 
 <h2 id="{{page.sections['general']['secs']['security'].anchor}}">{{page.sections['general']['secs']['security'].title}}</h2>
-Access to PiWeb Server service might require authentication. Authentication can be either *basic authentication*, based on username and password, or *Windows authentication* based on Active Directory integration.
+Access to PiWeb Server service might require authentication. Authentication can be either *basic authentication*, based on username and password or *Windows authentication* based on Active Directory integration
```

Comment:
Maybe replace the first 'or' with a comma to not have two 'or's in this sentence.

Reply:
erledigt